django/db/migrations/loader.py

```diff
@@ -6,11 +6,15 @@
 
 from django.apps import apps
 from django.conf import settings
+from django.db import connection as connection_
```

Comment:
When `makemigrations` is being run, `MigrationLoader` gets [instantiated](https://github.com/django/django/blob/master/django/core/management/commands/makemigrations.py#L76) with `connection` set to `None`. The `django.db.connection` is a proxy for accessing default `DatabaseWrapper`. So, it will only check the default database in the case of `makemigrations` and chosen with `--database` option, or default if not specified, in case of `migrate`. To check all the databases we can iterate over `django.db.connections`.